src/features/game/lib/processEvent.ts

```diff
@@ -23,19 +23,116 @@ import { BumpkinItem, ITEM_IDS } from "../types/bumpkin";
 import { MaxedItem } from "./gameMachine";
 
 export const MAX_INVENTORY_ITEMS: Inventory = {
+  ...(Object.keys(EXOTIC_CROPS) as InventoryItemName[]).reduce(
```

Comment:
If you use `getKeys` here you should be able to have type safety without casting.